Sources/PointFree/Stripe.swift

```diff
@@ -71,6 +71,18 @@ public struct Stripe {
 
   public typealias Cents = Tagged<Stripe, Int>
 
+  public struct Coupon: Codable {
+    public private(set) var id: Id
+    public private(set) var percentOff: Int?
```

Comment:
This should be `Int` but I don't want crashes. In reality it should be `Either<Cents, Percent>` for whether `amount_off` or `percent_off` is populated.